APE1.rst

```diff
@@ -145,14 +160,27 @@ An APE's status can
   rationales, so that new community members can understand why a decision was
   made.
 
-Modifications to an APE
-.......................
+* "Withdrawn": If the APE authors no longer feel that their proposal is feasible,
+  they can choose to withdraw the APE. A withdrawn APE is in effect equivalent
+  to a Rejected APE in that it will not be implemented.
```

Comment:
Due to #122 I'm adding this suggestion
```suggestion
  to a Rejected APE in that it will not be implemented.
* "Superseded" : An APE that was accepted but has since had become irrelevant
  or entirely replaced by some other APE or event can be given this status.
  Such status changes may not require the regular APE modification process if
  whatever caused this status already followed a discussion process at least
  as long as the APE discussion process.
```